src/Evolution/Systems/RadiationTransport/M1Grey/Sources.hpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// \cond
+class DataVector;
+
+namespace RadiationTransport {
+namespace M1Grey {
+
+/*!
+ * \brief Compute the curvature source terms for the flux-conservative
+ * grey M1 radiation transport.
+ *
+ *
+ * A flux-conservative system has the generic form:
```

Comment:
This my fault, but it is a flux-balanced system.  (Mathematicians reserve flux-conservative systems to have no sources)